src/clusterfuzz/_internal/bot/tasks/cron/sync_admins.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Cron to sync admin users."""
+
+from googleapiclient import discovery
+
+from clusterfuzz._internal.base import utils
+from clusterfuzz._internal.datastore import data_types
+from clusterfuzz._internal.datastore import ndb_utils
+from clusterfuzz._internal.metrics import logs
+
+
+def admins_from_iam_policy(iam_policy):
+  """Get a list of admins from the IAM policy."""
+  # Per
+  # https://cloud.google.com/appengine/docs/standard/python/users/adminusers, An
+  # administrator is a user who has the Viewer, Editor, or Owner primitive role,
+  # or the App Engine App Admin predefined role
+  roles = [
+      'roles/editor',
+      'roles/owner',
+      'roles/viewer',
+      'roles/appengine.appAdmin',
+  ]
+
+  admins = []
+  for binding in iam_policy['bindings']:
```

Comment:
Can we assume these fields always exist?

Reply:
It's from the exiting script and getIamPolicy() does return the bindings filed. https://cloud.google.com/storage/docs/json_api/v1/buckets/getIamPolicy